export-parameters/client.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+if [ "$1" = "client_initialization" ]
+then
+	# install build and runtime dependencies
+	yum -y install nfs-utils time
+	mkdir -p /mnt/ganesha
+
+	echo "------------------------------------------------------------------------"
+	echo "Client Initial Stage --- With All Rights To All Clients ( RO & RW ) "
+	echo "------------------------------------------------------------------------"
+	
+	#mount
+	mount -t nfs -o vers=3 ${SERVER}:${EXPORT} /mnt/ganesha
+	cd /mnt/ganesha
+	echo "Trying To Write A File"
+	echo "Hello World" > testFile.txt
+	ret=$?
+	if [ $ret -ne 0 ]
+	then
+		echo "FAILURE: Write permissions denied"
+		exit ret
```

Comment:
This `ret` seems wrong, just like in one of the other PRs.

Reply:
@nixpanic  Done